pages/form-field/permutations.page.tsx

```diff
@@ -36,6 +37,7 @@ const permutations = createPermutations<FormFieldProps>([
   },
   {
     label: ['Normal label'],
+    errorIconLabel: ['Error:'],
```

Comment:
Should it include the colon or not?

Reply:
You caught me being indecisive! I like the colon if the whole error message was one string, but I don't think it belongs in the label of an icon. Removing it.